pkg/resources/statefulsets/minio-statefulset.go

```diff
@@ -447,6 +447,15 @@ func NewForMinIOZone(t *miniov1.Tenant, wsSecret *v1.Secret, zone *miniov1.Zone,
 
 	if zone.VolumeClaimTemplate != nil {
 		pvClaim := *zone.VolumeClaimTemplate
+		if t.Spec.PurgePVCOnDeleteTenantCRD {
```

Comment:
if `purgePVCsOnDelete` is doing the same as this, and we call that on `processDelete` why do we need this here? seems like regular updates to the tenant won't remove the reference either

Reply:
This is to reflect the ownership rightaway